.github/workflows/publish.yml

```diff
@@ -0,0 +1,62 @@
+name: Publish Package to npm
+
+on:
+  workflow_dispatch:
+    inputs:
+      packageVersion:
+        description: "The version to publish (e.g. patch, minor, major)"
+        required: true
+        default: "patch"
+      distTag:
+        description: "The dist-tag to publish (e.g. latest, beta)"
+        required: true
+        default: "latest"
+
+jobs:
+  authorize:
+    name: Authorize
+    runs-on: ubuntu-latest
+    steps:
+      - name: ${{ github.actor }} permission check to do a release
+        uses: "lannonbr/repo-permission-check-action@2.0.2"
+        with:
+          permission: "write"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  
+  release:
+    name: Publish to Github Packages
+    runs-on: ubuntu-latest
+    needs: [authorize]
+    env:
+      PACKAGE_VERSION: ${{ github.event.inputs.packageVersion }}
+      DIST_TAG: ${{ github.event.inputs.distTag }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version-file: ".nvmrc"
```

Comment:
heads up that setup-node doesn't actually support everything that `.nvmrc` does; i suggest using my ljharb/node/install action instead.

Reply:
Did I miss something